dbt/models/marts/students/dim_student_activity.sql

```diff
@@ -0,0 +1,125 @@
+with 
+course_structure as (
+    select *
+    from {{ ref('dim_course_structure') }}
+),
+
+user_levels as (
+    select *
+    from {{ ref('dim_user_levels') }}
+),
+
+section_mapping as (
+    select *
+    from {{ ref('int_section_mapping') }}
+),
+
+school_years as (
+    select * 
+    from {{ ref('int_school_years') }}
+),
+
+school_status as (
+    select * 
+    from {{ ref('dim_school_status') }}
+),
+
+teacher_status as (
+    select * 
+    from {{ ref('dim_teacher_status') }}
+),
+
+schools as (
+    select * 
+    from {{ ref('dim_schools') }}
+),
+
+section_size as (
+    select 
+        section_id
+        , count(distinct student_id)                                as section_size
+    from {{ ref('int_section_mapping') }}
+    group by section_id
+)
+
+select 
+    ul.user_id                                                      as student_id
+    , ul.level_id                                                   as level_id
+    , ul.script_id                                                  as script_id
+    , date_trunc('day', ul.created_dt)                              as activity_date
+
+    -- time variables 
+    , date_trunc ('month', ul.created_dt)                           as activity_month  
+    , case 
+        when extract ('month' from ul.created_dt) in (7,8,9) then 'Q1'
+        when extract ('month' from ul.created_dt) in (10,11,12) then 'Q2'
+        when extract ('month' from ul.created_dt) in (1,2,3) then 'Q3'
+        when extract ('month' from ul.created_dt) in (4,5,6) then 'Q4'
+      end                                                           as school_year_quarter
+    , sy.school_year                                                as activity_school_year
+
+    -- course, script, unit, lesson, and level characteristics
+    , cs.level_name                                                 as level_name
+    , cs.level_type                                                 as level_type
+    , cs.script_name_long                                           as script_name
```

Comment:
```suggestion
    , cs.script_name                       as script_name
    , cs.unit
```
In `course_structure` script_name_long and script_name_short are fields that were manually updated through the previous year, but we shouldn't use them anymore. Unit is the equivalent,  and it is parsed from the script name